r2d7/cardlookup.py

```diff
@@ -12,6 +12,7 @@
 class CardLookup(DroidCore):
     def __init__(self):
         super().__init__()
+        self.register_handler(r'\{\{(.*)\}\}', self.handle_image_lookup)
         self.register_handler(r'\[\[(.*)\]\]', self.handle_lookup)
         self.register_dm_handler(r'(.*)', self.handle_lookup)
```

Comment:
This DM handler means that if you DM the bot with {{}}, you get a card lookup. If you register a dm handler with the {{}} pattern, you can fix that, but you'll need to change `DroidCore._dm_handlers` to be an `OrderedDict` so that the random order of dict iterators doesn't decide which handler gets called.

Reply:
Done. It makes the API slightly weird but it's the only solution I could think of (apart from stricter regexps) when I hit the same problem on my bot.

I've changed the non-dm handler to an `OrderedDict` as well so that both handlers have the same behaviour (otherwise some future dev like myself will get properly confused)